packages/e2e-tests/specs/performance/post-editor.test.js

```diff
@@ -124,24 +160,25 @@ describe( 'Post Editor Performance', () => {
 
 		// Measuring typing performance
 		await insertBlock( 'Paragraph' );
-		i = 200;
+		i = 20;
 		await page.tracing.start( {
 			path: traceFile,
 			screenshots: false,
 			categories: [ 'devtools.timeline' ],
 		} );
 		while ( i-- ) {
+			// Wait for the browser to be idle before starting the monitoring.
+			// eslint-disable-next-line no-restricted-syntax
+			await page.waitForTimeout( 200 );
```

Comment:
I noticed that by adding these timeouts between measures, it makes the numbers more stable, the reason is simple I think: we do a lot of lazy work when the browser is idle, if we don't give time for the browser to perform this kind of work, it's going to impact the intermediary typing measures (same for the other measures). So I'm hoping this PR will make the numbers a bit more stable.